aima-core/src/main/java/aima/core/nlp/ranking/LinkFinder.java

```diff
@@ -0,0 +1,25 @@
+package aima.core.nlp.ranking;
+
+import java.util.Hashtable;
+import java.util.List;
+
+public interface LinkFinder {
+	
+	/**
+	 * Take a Page object and return its outlinks as a list of strings.
+	 * The Page object must therefore possess the information to determine
+	 * what it links to.
+	 * @param page
+	 * @return
+	 */
+	public List<String> getOutlinks( Page page );
+	
```

Comment:
There is no need to define 'public' on any methods or attributes on an interface as they are public by default.